control-plane-operator/controllers/hostedcontrolplane/autoscaler/reconcile.go

```diff
@@ -284,3 +289,61 @@ func ReconcileAutoscaler(ctx context.Context, c client.Client, hcp *hyperv1.Host
 
 	return nil
 }
+
+const BalancingIgnoreLabelArg = "--balancing-ignore-label"
+
+// The following values are for cloud providers which have not yet created specific nodegroupset processors.
+// These values should be removed and replaced in the event that one of the cloud providers creates a nodegroupset processor.
+
+// AWS cloud provider ignore labels for the autoscaler.
+const (
+	// AwsIgnoredLabelEksctlNodegroupName is a label used by eksctl to identify "node group" names.
+	AwsIgnoredLabelEksctlNodegroupName = "alpha.eksctl.io/nodegroup-name"
```

Comment:
we don't need any of the EKS labels. Nothing will ever set those in an openshift cluster.